bungee/src/main/java/me/saiintbrisson/bungee/command/command/BungeeCommand.java

```diff
@@ -211,7 +212,9 @@ public BungeeCommand createRecursive(String name) {
 
     @Override
     public List<String> getAliasesList() {
-        return Arrays.asList(getAliases());
+        val aliases = this.getAliases();
+        if(Objects.isNull(aliases)) return Lists.newArrayList();
+        return Arrays.asList(aliases);
```

Comment:
Eu prefiro não usar o `val` do lombok aqui. E como essa lista não é mutável, podemos usar o `emptyList`.

```suggestion
        return aliases == null ? Collections.emptyList() : Arrays.asList(aliases);
```